wordcloud.py

```diff
@@ -57,41 +55,41 @@ def process_string(text_to_process):
 
 def determine_occurrences(list_data):
     """Add list to dictionary with value based on occurrence."""
-    word_to_occurrence = {}
+    word_to_count = {}
     for word in list_data:
-        word_to_occurrence[word] = list_data.count(word)
-    return word_to_occurrence
+        word_to_count[word] = list_data.count(word)
+    return word_to_count
 
 
 def allocate_values(list_data):
     """Add list to dictionary with value based on nominated keywords."""
-    word_to_occurrence = {}
+    word_to_count = {}
     for word in list_data:
-        word_to_occurrence[word] = 1
+        word_to_count[word] = 1
         if word in WORDS_WITH_VALUE:
-            word_to_occurrence[word] = (list_data.count(word) * 5)
-    return word_to_occurrence
+            word_to_count[word] = (list_data.count(word) * 5)
+    return word_to_count
 
 
 def determine_length(list_data):
```

Comment:
The name could be misleading. It sounds like it's determining the length of the given data. Instead, it maps words to word lengths.

Reply:
OK thank you.  Yes, again I've changed the name of it to create_dictionary_based_on_word_length().